redpanda/templates/statefulset.yaml

```diff
@@ -69,7 +69,11 @@ spec:
               fi;
               rpk --config $CONFIG config set redpanda.advertised_rpc_api.address $SERVICE_NAME;
               rpk --config $CONFIG config set redpanda.advertised_rpc_api.port {{ .Values.config.redpanda.rpc_server.port }};
+              {{- if .Values.loadBalancer.enabled }}
+              rpk --config $CONFIG config set redpanda.advertised_kafka_api.address {{ template "redpanda.name" . }}-$NODE_ID.{{.Values.loadBalancer.parentZone}}
```

Comment:
I'll be adding a fix for this shortly, but I believe that this should be `redpanda.fullname` in case "redpanda" is not used for the `Release.Name`.